TESTS/netsocket/tcp/tcpsocket_echotest.cpp

```diff
@@ -100,27 +100,25 @@ void tcpsocket_echotest_nonblock_receive()
         int recvd = sock.recv(&(tcp_global::rx_buffer[bytes2recv_total - bytes2recv]), bytes2recv);
         if (recvd == NSAPI_ERROR_WOULD_BLOCK) {
             if (tc_exec_time.read() >= time_allotted) {
-                TEST_FAIL();
                 receive_error = true;
```

Comment:
Not failure anymore if the test runs out of time?

Reply:
You're right - if the time alloted is exceeded, we will not report a failure any more. I wonder if I should bring it back in here or perhaps add it to line 193:
```
        if (receive_error) {
            break;
        }
```
Personally, I think the latter would be cleaner.